tests/virt_autotest/hotplugging_utils.pm

```diff
@@ -71,7 +71,7 @@ sub get_disk_image_name {
 sub reset_guest {
     my $guest = shift;
     return if $guest == "";
-    my $guest_instance = $virt_autotest::common::guests{$guest};
+    #    my $guest_instance = $virt_autotest::common::guests{$guest};
```

Comment:
It is better to delete unnecessary space or delete this line.

Reply:
yes